packages/utils/upgrade/src/modules/codemod-repository/__tests__/repository.test.ts

```diff
@@ -0,0 +1,169 @@
+import { vol, fs } from 'memfs';
+
+import { Range, SemVer } from 'semver';
+import {
```

Comment:
I have a doubt all of a sudden, but aren't we supposed to import those after mocking fs?

Reply:
why? 😅